src/analyze.cpp

```diff
@@ -2247,8 +2247,10 @@ static Error resolve_struct_type(CodeGen *g, ZigType *struct_type) {
     if (first_packed_bits_offset_misalign != SIZE_MAX) {
         size_t full_bit_count = packed_bits_offset - first_packed_bits_offset_misalign;
         size_t full_abi_size = get_abi_size_bytes(full_bit_count, g->pointer_size_bytes);
-        next_offset = next_field_offset(next_offset, abi_align, full_abi_size, abi_align);
-        host_int_bytes[gen_field_index] = full_abi_size;
+        size_t full_size = packed ? get_store_size_bytes(full_bit_count) : full_abi_size;
```

Comment:
Is there ever a time this conditional will be entered that we aren't dealing with a packed struct?